[android][test_app] Nightly dimension, input shape in gradle

IvanKobzarev opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#30195 [android][test_app] Nightly dimension, input shape in gradle**

1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMbqNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testappt/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

Differential Revision: [D18893394](https://our.internmc.facebook.com/intern/diff/D18893394)

3. mbqCamera - CameraActivity testing with ImageNet classes

IvanKobzarev committed:
[android][test_app] Nightly dimension, input shape in gradle

IvanKobzarev committed:
Update on "[android][test_app] Nightly dimension, input shape in gradle"


1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMobNet2QuantNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testapp.mobNet2Quant/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

IvanKobzarev committed:
Update on "[android][test_app] Nightly dimension, input shape in gradle"


1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMobNet2QuantNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testapp.mobNet2Quant/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

Differential Revision: [D18893394](https://our.internmc.facebook.com/intern/diff/D18893394)

IvanKobzarev committed:
Update on "[android][test_app] Nightly dimension, input shape in gradle"


1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMobNet2QuantNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testapp.mobNet2Quant/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

Differential Revision: [D18893394](https://our.internmc.facebook.com/intern/diff/D18893394)

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit d67f312:


* **1/2** broken upstream at merge base 8f3c0d5 (see [grid view](https://dr.pytorch.org/master-timeline.html?count=1&sha1=8f3c0d541eeaf7316d2e62f98cb1337080144206&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35))
  <details>
  <summary>
  You may want to rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  
* **1/2** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=d67f31279d156a86d57fc8987e43b27f9bd6da61).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably caused by upstream breakages:
* [pytorch_macos_10_13_py3_test](https://dr.pytorch.org/build-details.html?build_id=77959507)

### 1 failure *not* recognized by patterns:
| Job | Step | Status |
| --- | --- | --- |
| ![CircleCI](https://avatars0.githubusercontent.com/ml/7?s=12) <sup>pytorch-linux-xenial-py3-clang5-android-ndk-r19c-gradle-build-x86_32</sup> | <sup>pytorch android gradle build only x86_32 (for PR)</sup> | New in PR |


---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

IvanKobzarev committed:
Update on "[android][test_app] Nightly dimension, input shape in gradle"


1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMbqNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testappt/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

Differential Revision: [D18893394](https://our.internmc.facebook.com/intern/diff/D18893394)

3. mbqCamera - CameraActivity testing with ImageNet classes

IvanKobzarev committed:
Update on "[android][test_app] Nightly dimension, input shape in gradle"


1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMbqNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testappt/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

Differential Revision: [D18893394](https://our.internmc.facebook.com/intern/diff/D18893394)

3. mbqCamera - CameraActivity testing with ImageNet classes

IvanKobzarev committed:
Update on "[android][test_app] Nightly dimension, input shape in gradle"


1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMbqNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testappt/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

Differential Revision: [D18893394](https://our.internmc.facebook.com/intern/diff/D18893394)

3. mbqCamera - CameraActivity testing with ImageNet classes

IvanKobzarev committed:
Update on "[android][test_app] Nightly dimension, input shape in gradle"


1. Added flavorDimensions 'build' local/nightly
to be able to test the latest nightlies 

```
cls && gradle clean test_app:installMbqNightlyDebug -PABI_FILTERS=x86 --refresh-dependencies && adb shell am start -n org.pytorch.testappt/org.pytorch.testapp.MainActivity
```

 2. To be able to change all new model setup editing only `test_app/build.gradle`
 Inlined model asset file names to `build.gradle`

Extracted input tensor shape to `build.gradle` (BuildConfig)

Differential Revision: [D18893394](https://our.internmc.facebook.com/intern/diff/D18893394)

3. mbqCamera - CameraActivity testing with ImageNet classes

[ghstack-poisoned]
